psc/Main.hs

```diff
@@ -78,6 +78,20 @@ compile (PSCMakeOptions inputGlob inputForeignGlob outputDir opts usePrefix) = d
             hPutStrLn stderr (P.prettyPrintMultipleWarnings (P.optionsVerboseErrors opts) warnings')
           exitSuccess
 
+globWarningOnMisses :: String -> [FilePath] -> IO [FilePath]
+globWarningOnMisses fileType = go ([], [])
+  where
+  go (hits, misses) [] = do
+    unless (null misses) $ do
+      let formattedMisses = tail $ init $ show misses
+      hPutStrLn stderr $ "psc: No " ++ fileType ++ " found using: " ++ formattedMisses
```

Comment:
Instead of piecing strings together, maybe take a function with throws the warning?